### ISSUE TITLE
3.5.3 - not possible to deactivate user which is room owner

### ISSUE THREAD

**Issue by emikolajczak** [RocketChat/Rocket.Chat#18659]
### Description:

There is not possible to "Deactivate" user which is owner of any room. From admin panel, when click deactivate dialaog box appears but is greyed out and "Yes, deactivate it!" can not be clicked.

![image](https://user-images.githubusercontent.com/50440047/91035147-6e1fee00-e606-11ea-84d5-045bba7b9964.png)

Also it is impossible from REST API. Bellow error appears

```
  "details": {
    "shouldBeRemoved": [
      "test trim"
    ],
    "shouldChangeOwner": [
      "archiwum"
    ]
  },
  "error": "[user-last-owner]",
  "errorType": "user-last-owner",
  "success": false
} 
```

This bug totally blocks us from upgrade to newest version.

### Steps to reproduce:

1. Go to Admin -> users and search for user which is owner of any room
2. Clikc "Deactivate"
3. Greyed out dialog box appears, user cannot be deactivated
4. Try "Deactivate" user from REST API call
5. Described error appear

### Expected behavior:

User will be deactivated and new owners of rooms will be assigned

### Actual behavior:

Not possible to deactivate an user

### Server Setup Information:

- Version of Rocket.Chat Server: 3.5.3
- Operating System: Centos 7
- Deployment Method: docker-compose
- Number of Running Instances: 9
- DB Replicaset Oplog: YES
- NodeJS Version: v12.16.1
- MongoDB Version: 4.0.14

### Client Setup Information

- Desktop App or Browser Version: both
- Operating System: Windows 10

**Comment by emikolajczak** [comment]
Still an issue on 3.5.4

**Comment by gabriellsh** [comment]
Closed by #18596 

Please, make sure to do a quick search on the open & closed issues before opening a new one. I closed a few of these already. Thanks for your comprehension 😉 !

**Comment by emikolajczak** [comment]
Hi, thanks for fix.
Is this fix also for REST API?
